cmd/root_test.go

```diff
@@ -15,7 +15,7 @@ var _ = Describe("Root", func() {
 	Describe("subcommands", func() {
 		It("can create commands", func() {
 			commands := cmd.MakeCommands()
-			Expect(len(commands.Commands())).To(Equal(17))
+			Expect(len(commands.Commands())).To(Equal(18))
```

Comment:
maybe replace this with a golden test on `--help`?

Reply:
Not sure I wanna start re-doing all of the CLI's tests just now!

Reply:
I've put in a bunch of usage test assertions for the runner commands.

Once this merges, I'll have a look at how hard it'd be to do the same for all the commands in the CLI.

Reply:
I was thinking just one that lists all of the commands - but I had forgotten that there isn't a help command which lists all commands, and these are hidden anyway!

Reply:
Good point!